packages/hydrogen/src/customer-privacy/ShopifyCustomerPrivacy.tsx

```diff
@@ -1,7 +1,7 @@
 import {useLoadScript} from '@shopify/hydrogen-react';
 import {useEffect, useRef} from 'react';
 
-export type ConsentStatus = 'true' | 'false' | '';
+export type ConsentStatus = boolean | undefined;
```

Comment:
You might need to double check if `window.Shopify.customerPrivacy` actually accepts boolean. At least according to the shopify doc, it takes `'true' | 'false'` and defaults to empty string representing as unset

Reply:
nvm .. just saw the comment